app/controllers/application_controller.rb

```diff
@@ -13,4 +13,19 @@ def require_organiser
       redirect_to root_path, alert: "Login again as a organiser!"
     end
   end
-end
\ No newline at end of file
+
+  def require_mentee
+    unless current_user && current_user.mentee?
+      redirect_to root_path, alert: "Login again as a Mentee!"
+    end
+  end
+ 
+  def require_mentee_or_organiser
```

Comment:
i think we can add it only in mentee_profiles_controller as we're not gonna use it anywhere else